src/main/scala/strawman/collection/Map.scala

```diff
@@ -0,0 +1,44 @@
+package strawman.collection
+
+import strawman.collection.mutable.Builder
+
+import scala.Option
+import scala.annotation.unchecked.uncheckedVariance
+import scala.Predef.???
+
+/** Base Map type */
+trait Map[K, +V]
+  extends Iterable[(K, V)]
```

Comment:
I think I’m going to learn something here: I thought `(K, V)` was a syntactic shortcut for `Tuple2[K, V]`, is it?